lib/atomic-management.js

```diff
@@ -528,12 +534,28 @@ export default {
     const message = 'Invalid Package Names';
     const description =
       'The following packages are configured in the local project ' +
-      'configuration file but are invalid packages. Please fix your config file.\n\n' + 
+      'configuration file(s) but are invalid packages. Would you like to ' +
+      'open the configuration file(s) to fix this?\n\n' +
       packageNames.map(name => `- ${name}`).join('\n');
 
     const notification = atom.notifications.addWarning(message, {
       description,
-      dismissable: true
+      dismissable: true,
+      buttons: [
+        {
+          text: 'Open Configuration File(s)',
+          onDidClick: () => {
+            this.openAllConfigFiles();
```

Comment:
This will open all the config files, don't you wanna open just the ones to fix it